app/Http/Middleware/MaintenanceMode.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Ushahidi\App\Http\Middleware;
+
+use Closure;
+
+class MaintenanceMode
+{
+
+   /**
+    * Handle an incoming request.
+    *
+    * @param  \Illuminate\Http\Request  $request
+    * @param  \Closure  $next
+    * @param  string|null  $guard
+    * @return mixed
+    */
+    public function handle($request, Closure $next, $guard = null)
+    {
+        $maintenanceMode = getenv('MAINTENANCE_MODE');
```

Comment:
It's probably better to use laravel's `env('MAINTENANCE_MODE', false)` helper. It also handles turning 'true' in a boolean true.
https://laravel.com/docs/5.6/configuration#environment-variable-types